packages/oruga-next/src/components/tooltip/Tooltip.vue

```diff
@@ -161,6 +161,11 @@ const props = defineProps({
         type: [String, Array, Function] as PropType<ComponentClass>,
         default: undefined,
     },
+    /** Root element class indicated dropdown position */
+    rootPositionClass: {
```

Comment:
I'm not fine with the rootPositionClass on the Tooltip component. I think it is not needed. The position is a feature which is implemented by theme itself and not by the bulma library. So it should be possible to refactor the theme to use the actuall positionClass. The bootstrap theme did the same.

Reply:
fair enough. removed.